docs/opensanctions-delivery.md

```diff
@@ -0,0 +1,20 @@
+# Configuring OpenSanctions Data
+
+By default, yente is configured to access the OpenSanctions database. We ask prospective commercial customers to create a Data Delivery Token to access the data. Please register at our [customer portal]({{ config.extra.opensanctions_url }}/account/) and find your Delivery Token in the Credentials Manager. Set `OPENSANCTIONS_DELIVERY_TOKEN` in your environment and off you go!
```

Comment:
By default, yente is configured to fetch the full, up-to-date OpenSanctions database. To help us understand how commercial partners use the dataset, we ask commercial customers to create a delivery token. The token itself is not paid, and helps us to provide relevant support. Please sign up at the [customer portal]({{ config.extra.opensanctions_url }}/account/) - a delivery token will be created automatically and immediately. Set `OPENSANCTIONS_DELIVERY_TOKEN` in your environment and off you go!